Makefile

```diff
@@ -1,88 +1,114 @@
 testable_packages=$(shell go list ./... | egrep -v 'constants|mocks|testing')
 project=$(shell basename $(PWD))
-project_test=${project}-test
-pg_dep=$(project)_postgres_1
-test_packages=`find . -type d -name "docker_data" -prune -o \
-							-type f -name "*.go" ! \( -path "*vendor*" \) -print \
-							| sed -En "s/([^\.])\/.*/\1/p" | uniq`
-database=postgres://postgres:$(project)@localhost:8432/$(project)?sslmode=disable
-database_test=postgres://postgres:$(project)@localhost:8432/$(project_test)?sslmode=disable
-platform=darwin
+test_db_name=${project}-test
+pg_docker_image=$(project)_postgres_1
+db_url=postgres://postgres:$(project)@localhost:8432/$(project)?sslmode=disable
+db_test_url=postgres://postgres:$(project)@localhost:8432/$(test_db_name)?sslmode=disable
+uname_S=$(shell uname -s)
+
+# TravisCI runs in Linux instances, while developers run in MacOS machines
+ifeq ($(uname_S), Darwin)
+  platform := darwin
+else
+  platform := linux
+endif
 
 export GO111MODULE=on
 
-setup: setup-project setup-deps
+.PHONY: all
+all: setup-migrate download-mod db-setup
 
-setup-project:
-	@go mod download
+.PHONY: all-ci
+all-ci: setup-migrate download-mod db-setup-test
```

Comment:
Suggestion: ci/install instead of all-ci. Same for ci/test vs test-ci. (I'd rather have a script, though.)